src/pybritive/britive_cli.py

```diff
@@ -474,8 +476,8 @@ def checkout(self, alias, blocktime, console, justification, mode, maxpolltime,
 
         # these 2 modes implicitly say that console access should be checked out without having to provide
         # the --console flag
-        if mode and (mode == 'console' or mode.startswith('browser')):
-            console = True
+        if console := (mode and (mode == 'console' or mode.startswith('browser'))):
```

Comment:
assignment expressions are supported in Python 3.8 and up. The SDK/CLI support 3.7 and up. 

https://peps.python.org/pep-0572/